shared/src/androidMain/kotlin/com/fbada006/shared/utils/ImagePickerFactory.android.kt

```diff
@@ -0,0 +1,17 @@
+package com.fbada006.shared.utils
+
+import androidx.activity.ComponentActivity
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalContext
+
+actual class ImagePickerFactory {
+
+    @Composable
+    actual fun createPicker(): ImagePicker {
```

Comment:
I believe if you use an actual/expected on the class, you can eliminate the actual/expected on the methods. It is kind of like an interface. The methods will be provided as the instance methods on a type are always available.

Reply:
I understand this but expect/actual classes are a bit different with interfaces with expect/actual functions. 
With classes, we need to define the methods with actual. If we eliminate, there will be compile errors. 
Check this for [reference](https://kotlinlang.org/docs/multiplatform-expect-actual.html#expected-and-actual-classes)